.woodpecker/test.yml

```diff
@@ -5,22 +5,21 @@ when:
   - event: pull_request
 
 steps:
-  lint:
-    image: alpine/helm:4.0.1
+  - name: lint
+    image: docker.io/alpine/helm:4.0.1
     commands:
       - helm lint --with-subcharts charts/*
 
-  unittest:
-    image: alpine/helm:4.0.1
+  - name: unittest
+    image: docker.io/alpine/helm:4.0.1
     environment:
-      # NB: we can't use the official helmunittest/helm-unittest image as it's rootless and WP does not support that
```

Comment:
unrelated removal

Reply:
No. We dont want to use the plugin image to ensure we use the same helm version everywhere, so removed it.